commit 0fa652c680eb3cb3d0175a587df6bddf76463e60 (refs/changes/95/4795/3)
Author: Richard Kiene <richard.kiene@joyent.com>
Date:   2018-10-12T20:58:28+00:00 (12 months ago)
    
    TRITON-755 Want ability to filter CMON discovery results by tag
    
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index 040d4bc..9fd8b77 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "sdc-docker",
-  "version": "0.5.3",
+  "version": "0.5.4",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -28,7 +28,7 @@
     "strsplit": "1.0.0",
     "tape": "^4.4.0",
     "trace-event": "1.2.0",
-    "triton-tags": "1.1.4",
+    "triton-tags": "1.2.2",
     "ufds": "1.2.0",
     "vasync": "2.1.0",
     "verror": "1.9.0",
